src/main/resources/view/DarkTheme.css

```diff
@@ -90,34 +90,36 @@
 .list-view {
     -fx-background-insets: 0;
     -fx-padding: 0;
-    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-background-color: #A3CEA2;
+    -fx-border-color: #457C51;
 }
 
 .list-cell {
     -fx-label-padding: 0 0 0 0;
     -fx-graphic-text-gap : 0;
     -fx-padding: 0 0 0 0;
+    -fx-text-fill: black;
 }
 
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: #B5C89C;
```

Comment:
Does JavaFX have support for CSS custom properties? It might be better to define the color scheme using that instead.

Reply:
Great idea~